mono/mini/mini-runtime.c

```diff
@@ -4080,6 +4080,8 @@ mini_init (const char *filename, const char *runtime_version)
 
 	CHECKED_MONO_INIT ();
 
+	mono_threads_suspend_policy_init ();
+
```

Comment:
Shouldn't this be called from like mono_thread_info_init () ?

Reply:
Yea makes sense. I'll move it.